common/flatpak-dir.c

```diff
@@ -7584,6 +7584,15 @@ export_desktop_file (const char         *app,
       /* Add a marker so consumers can easily find out that this launches a sandbox */
       g_key_file_set_string (keyfile, G_KEY_FILE_DESKTOP_GROUP, "X-Flatpak", app);
 
+      /* Disable krunner dbusplugins by default, so that flatpak applications cannot
+       * unintentionally grab sensible search data.
+       */
+      if (g_key_file_get_boolean (keyfile, G_KEY_FILE_DESKTOP_GROUP,
+                               "X-KDE-PluginInfo-EnabledByDefault", NULL))
+        {
+          g_key_file_set_string (keyfile, G_KEY_FILE_DESKTOP_GROUP, "X-KDE-PluginInfo-EnabledByDefault", "false");
+        }
```

Comment:
```suggestion
      g_key_file_set_boolean (keyfile, G_KEY_FILE_DESKTOP_GROUP, "X-KDE-PluginInfo-EnabledByDefault", FALSE);
```

Reply:
Your suggestion would place the key in every _desktop_ file. The idea was to keep that more clean and only set it to _false_, if there is such a key and has the value _true_. I guess we don't want to place that key in all desktop files.

Reply:
That was my intention, I find explicitly setting it more clear. However it’s not necessary.

The other change was using set_boolean.

Reply:
Ah, good point. I didn't recognize the set_boolean at the first glance. Just updated it. If it's OK for you, I'd leave the "only-if-defined-true" logic in there, because that's a bit special for the runner desktop files - while not being relevant for ordinary desktop files.